### PR TITLE
Mpe room creation connect to client

### LINKED ISSUE #288: Connect client to MPE room creation funnel
Client:
 - [x] Create a socketConnection Service to listen to every MPE room related server socket event in the MPE actor machine wrapper
 - [x] On MPE_ROOM_CREATE_SYNCED_CALLBACK spawn a new actor with the given state
 - [ ] On MPE_ROOM_CREATE_CALLBACK unlock the MPE room view and merge the given state
 - [ ] Create a service that will manage each received event and merge the given state in the involved spawned actor only, something like a proxy

### PR DESCRIPTION
**Pull request by prastoin**
close #288

### CODE REVIEW

**Review by Devessier — CHANGES_REQUESTED** [review]
It's so nice! Our MPE feature starts being alive from end-to-end!
Let's build some more things and the project will be finished.
Great job!

Before merging, I think some minor improvements should be made.

File: packages/client/machines/appMusicPlaylistsMachine.ts [hunk 1]
```diff
@@ -57,19 +80,73 @@ export function createAppMusicPlaylistsMachine({
...
+        on: {
+            FORWARD_ASSIGN_MERGE_NEW_STATE_TO_INVOLVED_PLAYLIST: {
+                actions: send(
+                    (_, { state }) => ({
+                        type: 'ASSIGN_MERGE_NEW_STATE',
+                        state,
+                    }),
+                    {
+                        to: (_, { state, type }) => {
+                            //Add on error create mpe bool ?
```
[the remark this answers is not in the document]
Comment (prastoin) [reply packages/client/machines/appMusicPlaylistsMachine.ts:138]: I mean that there's a very tiny race condition risk here, where the client receives the CREATE_MPE_ROOM_CALLBACK before the CREATE_MPE_SYNCED_CALLBACK
The actor wouldn't be already spawned in the machine

**Review by Devessier — APPROVED** [review]
It's perfect!
We can merge.

**Review by Devessier — DISMISSED** [review]
Even after rebase, it's still perfect.
We can merge.

File: packages/client/machines/appMusicPlaylistsMachine.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Devessier) [reply packages/client/machines/appMusicPlaylistsMachine.ts:142]: Detailed in https://github.com/AdonisEnProvence/MusicRoom/issues/292.

**Pull request merged**